android/src/main/java/org/mozilla/mozstumbler/client/serialize/ObservationPointSerializer.java

```diff
@@ -48,7 +49,7 @@
     private static final String MLS_NAME = "MLS";
     private static final String ICON_RED_CIRCLE = "http://maps.google.com/mapfiles/kml/shapes/placemark_circle_highlight.png";
     private static final String STYLE_NAME_RED_CIRCLE = "redcircle";
-    private static final String ICON_ARROW = "http://maps.google.com/mapfiles/kml/shapes/arrow.png";
+    private static final String ICON_ARROW = "http://earth.google.com/images/kml-icons/track-directional/track-0.png";
```

Comment:
I don't know if we should use those. MozStumbler does not load these icons. The only other app I used was Google Earth, which does change the icon based on these links, but it already has them available locally. It could even do some magic:
> Note: The track icons are treated special in Google Earth. If you specify to use the gx:Track extension as used for GPS-tracks and specify a Style using the http://earth.google.com/images/kml-icons/track-directional/track-0.png icon (shown above) then the icon is dynamically changed to appropriate icon using the computed compass heading from the previous position.

I found the information and icon URL here: http://kml4earth.appspot.com/icons.html